output/cloud/output.go

```diff
@@ -200,18 +202,36 @@ func (out *Output) Start() error {
 		out.config = out.config.Apply(*response.ConfigOverride)
 	}
 
-	out.startBackgroundProcesses()
+	err = out.startBackgroundProcesses()
+	if err != nil {
+		return err
+	}
 
 	out.logger.WithFields(logrus.Fields{
 		"name":        out.config.Name,
 		"projectId":   out.config.ProjectID,
 		"duration":    out.duration,
 		"referenceId": out.referenceID,
+		"apiVersion":  out.config.APIVersion.Int64,
 	}).Debug("Started!")
 	return nil
 }
 
-func (out *Output) startBackgroundProcesses() {
+func (out *Output) startBackgroundProcesses() error {
+	if out.config.APIVersion.Int64 == 2 {
```

Comment:
I know that is obvious, and we probably don't have many versions, but I suggest avoiding using the "magic" number 2 and creating a constant. 

That way, we keep code more maintainable in the future since any other maintainer/contributor will easily find all such conditional places and won't spend (or miss) the places where we compare with some number.

Reply:
This is resolved now on the [root PR](https://github.com/grafana/k6/pull/3072) in the https://github.com/grafana/k6/pull/3072/commits/448da4c3217ccff1b21c07462724e13e628bc075 commit